### PR TITLE
pkg: add pt_BR translation

### PR DESCRIPTION
**Pull request by git-em**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/android/pkg.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Gerenciador de pacotes para o Termux.
+> Mais informações: <https://wiki.termux.com/wiki/Package_Management>.
+
+- Atualiza todos os pacotes instalados:
+
+`pkg upgrade`
+
+- Instala um pacote:
+
+`pkg install {{package}}`
```
Comment (navarroaxel) [inline pages.pt_BR/android/pkg.md:12]: Translate placeholders for non-English speakers
```suggestion
`pkg install {{pacote}}`
```

**Comment by git-em** [comment]
Thanks for the review and corrections!

**Comment by navarroaxel** [comment]
> Thanks for the review and corrections!

@git-em but you should make the change in every example :joy:

**Review by navarroaxel — APPROVED** [review]

**Comment by git-em** [comment]
😅

**Review by blueskyson — APPROVED** [review]

**Pull request merged**